File: src/PaperKiller/PaperKiller/DTO/StudentDTO.cs
namespace PaperKiller.DTO
{
    public class StudentDTO
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PhoneNumber { get; set; }
        public string CheckInDate { get; set; }
        public string Studak { get; set; }
        public string Gender { get; set; }
        public string RoomNumber { get; set; }
        public int? LinenId { get; set; }
        public int? ItemsId { get; set; }
    }

    public class StudentDTOAdd : StudentDTO
    {
        public string Login { get; set; }
        public string Password { get; set; }
        public string UserType { get; set; }
    }
}

File: src/PaperKiller/PaperKiller/wwwroot/app/src/interfaces/StudentDTO.ts
export interface StudentDTO {
    id: number;
    login: string;
    password: string;
    userType: string;
    name: string;
    surname: string;
    phoneNumber: string;
    checkInDate: string;
    studak: string;
    gender: string;
    roomNumber: string;
    linenId: number | null;
    itemsId: number | null;
}